helm/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 7.10.5
+version: 7.10.6
```

Comment:
Hi @jkroepke,

please update the artifact hub change entry at the end of this file.

Reply:
Done 👍